### ISSUE TITLE
Larceny writes #\esc instead of #\escape in R7RS mode

### ISSUE THREAD

**Issue by mnieper** [larcenists/larceny#778]
```
$ larceny -r7rs
Larceny v1.1a1 (Jul  1 2016 15:41:21, precise:Linux:unified)
larceny.heap, built on Fr 1. Jul 15:42:29 CEST 2016

> (import (scheme write))

> (write #\escape)
#\esc
> 
```

**Comment by mnieper** [comment]
Also, Larceny outputs `#\nul`, which is not defined by the R7RS.

**Comment by WillClinger** [comment]
Fixed by changeset c1176ea21e81c73d234e05318e872f429c39c2fe
